js/languages/en_US.json

```diff
@@ -1279,7 +1279,9 @@
       "dateAsc": "Newest",
       "dateDesc": "Oldest"
     },
-    "placeholderSearch": "Search by Order #, Title or Vendor",
+    "placeholderSearchsales": "Search by Order #, Title or Buyer",
```

Comment:
It's minor, but can we use proper camel-case here and then you could just use the `capitalize` utility function in `tab.html`?

Reply:
Will do.